tests/system/env-template.yml

```diff
@@ -44,3 +44,6 @@ MLRUN_SYSTEM_TESTS_KAFKA_BROKERS:
 
 # slack webhook for sending system test reports
 MLRUN_SYSTEM_TESTS_SLACK_WEBHOOK_URL:
+
+# sql db path string for test_sql_db - e.g. "mysql+pymysql://<username>:<password>@<host>:<port>/<db_name>
+MLRUN_SQL_DB_PATH_STRING:
```

Comment:
how is this propogated to runtimes?

Reply:
this is still not addressed - and so will not work with runtimes.
you should have a config entry in config.py, and then read that config from the code (since the env var will not always be available)